packages/components/src/theme/ComposerTheme/Components/index.ts

```diff
@@ -3,11 +3,13 @@ import { Spinner } from './Spinner';
 import { Tabs } from './Tabs';
 import { Text } from './Text';
 import { TextPairing } from './TextPairing';
+import { Badge } from './Badge';
 
 export const components = {
   Button,
   Spinner,
   Tabs,
   Text,
   TextPairing,
+  Badge,
```

Comment:
please follow an alphabetical order (yes, we should automate this 😉)

Reply:
Done!